.github/workflows/buildAndPublish.yml

```diff
@@ -0,0 +1,47 @@
+name: buildAndPublish
+
+on:
+  # Triggers the workflow on push
+  push:
+    branches: [ master ]
+
+  # Also trigger on release created events
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.ADMIN_PAT_TOKEN }}
```

Comment:
I don't remember if Github Actions shares secrets/config across repositories. Have you added this token in the Github UI?

Reply:
I don't actually know what this is. I figured that's because it's a secret thing...known to admins?